### DISCUSSION TITLE
Question about the license.

### DISCUSSION THREAD

**Discussion by smallfly** [NVlabs/instant-ngp#368] [Q&A] [answered]
Hello,

First, I want to say that this research is mind blowing. Thanks for sharing.

I wanted to ask if this is ok to share online (ex: social networks) renders/videos of models trained using Instant-ngp?

Thank you!

**Comment by Tom94** [comment, marked as answer]
Absolutely! And thank you for the kind words. :)

**Reply by smallfly** [reply]
Amazing!
